client/src/app/core/core-services/operator.service.ts

```diff
@@ -415,6 +416,24 @@ export class OperatorService {
         return result;
     }
 
+    public hasOrganizationPermissions(...permissionsToCheck: OML[]): boolean {
+        return permissionsToCheck.every(permission => (permsLookup.oml[this.OML] || 0) >= permsLookup.oml[permission]);
+    }
+
+    public hasCommitteePermissions(committeeId: Id, ...permissionsToCheck: CML[]): boolean {
+        // If a user can manage an entire organisation, he can also manage every committee.
+        const orgaPermissions = permissionsToCheck.includes(CML.can_manage) ? [OML.can_manage_organisation] : [];
```

Comment:
Is there a usecase for this swithc? I would expect to always check for OML.can_manage_organization or higher regardless of the CML requested.

Reply:
In the docs the permissions are described as follows: 
```
A user with superadmin can do everything: He is allowed to see non-personal models (e.g. not personal notes or password hashes of other users), but can enter every committee and meeting and is transitively manager of each committee and admin of each meeting.

A user with can_manage_organisation is indirectly a committee manager for every committee. The user can create, edit and delete committees and administrating resources and the motion forwarding structure.
```

Therefore, I checked this. But then, I can always check, which OML an operator has.